packages/react/src/__tests__/ReactContextValidator-test.js

```diff
@@ -225,6 +225,23 @@ describe('ReactContextValidator', () => {
     ReactTestUtils.renderIntoDocument(<Component testContext={{foo: 'foo'}} />);
   });
 
+  it('warns of incorrect prop types on context provider', () => {
+    const TestContext = React.createContext();
+
+    TestContext.Provider.propTypes = {
+      value: PropTypes.string.isRequired,
+    };
+
+    ReactTestUtils.renderIntoDocument(<TestContext.Provider value="val" />);
+
+    expect(() =>
+      ReactTestUtils.renderIntoDocument(<TestContext.Provider />),
+    ).toWarnDev(
+      'Warning: Failed prop type: The prop `value` is marked as required in ' +
+        '`Context.Provider`, but its value is `undefined`.',
```

Comment:
Please include stack in the expected message. You’ll want to use `**` placeholders similar to other tests that assert about the warning with stacks.

Reply:
@nicolevy you can find an example of what Dan means here: https://github.com/facebook/react/blob/master/packages/react/src/__tests__/ReactContextValidator-test.js#L182-L183